### PR TITLE
Add missing linux installation pages

### LINKED ISSUE #667: Document Memgraph install process for CentOS 7
Add missing documentation page for installing the CentOS 7

### LINKED ISSUE #668: Document Memgraph install process for Fedora
Add guide how to install Memgraph on Fedora.

### LINKED ISSUE #669: Document missing info how to install Memgraph on Rocky/Redhat
Add missing info how to install Memgraph on Rocky/Redhat

### PR DESCRIPTION
**Pull request by antejavor**
### Description

Add installation process for CentOS
Add installation process for Fedora 
Add installation process for Rocky 
Add installation process for Redhat 
Add installation process for Amazon Linux

Closes #667 
Closes #668 
Closes #669 

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/getting-started.mdx [hunk 1]
```diff
@@ -139,13 +139,29 @@ opt for the `memgraph` image if you prefer a database-only configuration.
...
+      title="Fedora"
+      href="/getting-started/install-memgraph/fedora"
+    />
+    <Card
+      title="Red Hat"
+      href="/getting-started/install-memgraph/redhat"
+    />
+    <Card
+      title="Rocky"
+      href="/getting-started/install-memgraph/Rocky"
```
Comment (katarinasupe) [inline pages/getting-started.mdx:155]:
```suggestion
      href="/getting-started/install-memgraph/rocky"
```

this is my assumption

File: pages/getting-started/install-memgraph.mdx [hunk 2]
```diff
@@ -47,6 +47,26 @@ Other installation options are also available.
...
+    title="Fedora"
+    href="/getting-started/install-memgraph/fedora"
+  />
+  <Card
+    title="Red Hat"
+    href="/getting-started/install-memgraph/redhat"
+  />
+  <Card
+    title="Rocky"
+    href="/getting-started/install-memgraph/Rocky"
```
Comment (katarinasupe) [inline pages/getting-started/install-memgraph.mdx:64]:
```suggestion
    href="/getting-started/install-memgraph/rocky"

```

again assumption

File: pages/getting-started/install-memgraph/amazon-linux.mdx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+You can also use [direct download
+links](/getting-started/install-memgraph/direct-download-links) to get the
+latest Memgraph packages. 
+
+For example for Amazon Linux 2:
+
+<CodeSnippet code="sudo wget https://download.memgraph.com/memgraph/v2.15.1/amzn-2/memgraph-2.15.1_1-1.x86_64.rpm" page="install-memgraph/amazon-linux" os="amazon-linux"/>
+
+{<h3>Install Memgraph</h3>}
+
```
Comment (katarinasupe) [inline pages/getting-started/install-memgraph/amazon-linux.mdx:40]: Please add note like on https://memgraph.com/docs/getting-started/install-memgraph/ubuntu#installation-guide (we had users reporting this):
For the latest version of Memgraph, replace the version in the download command with the one from the [direct download links](https://memgraph.com/docs/getting-started/install-memgraph/direct-download-links) page.

Comment (katarinasupe) [reply pages/getting-started/install-memgraph/amazon-linux.mdx:40]: same goes for other OS

File: pages/getting-started/install-memgraph/amazon-linux.mdx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+3.8.2 (default, Jul 16 2020, 14:00:26)
+[GCC 9.3.0]
+```
+
+If you wish to work with built-in NetworkX modules in Memgraph, you need to
+install the following Python libraries:
+* [NumPy](https://numpy.org/)
+* [SciPy](https://www.scipy.org/)
+* [NetworkX](https://networkx.org/)
+
```
Comment (katarinasupe) [inline pages/getting-started/install-memgraph/amazon-linux.mdx:116]: Is this still true? I don't have Linux, so just checking here. I guess it's a bit different in Docker

**Review by katarinasupe — APPROVED** [review]

**Review by kgolubic — CHANGES_REQUESTED** [review]
In new docs you have section that looks like:

```
## Installation guide

  <Steps>

  {<h3>Download Memgraph RPM Package</h3>}

Downloaded the latest Memgraph RPM Package from the [Memgraph Download
Hub](https://memgraph.com/download/).

You can also use [direct download
links](/getting-started/install-memgraph/direct-download-links) to get the
latest Memgraph packages. 


```

To me it makes no sense to have a section that talks about generic case (Download Memgraph RPM Package) and mentions `Downloaded the latest Memgraph RPM Package from the [Memgraph Download Hub](https://memgraph.com/download/).` when the page talks about specific Linux distribution.

And this part is mentioned, e.g. "`For example for Amazon Linux 2:`".

I think that it would make sense to remove this generic part about RPM and talk aonly about each distro in the individual documentation, e.g.

```
## Installation guide

  <Steps>

  {<h3>Download Memgraph for Amazon Linux</h3>}

Downloaded the latest Memgraph Package for Amazon Linux from the [Memgraph Download
Hub](https://memgraph.com/download/).

You can also use [direct download
links](/getting-started/install-memgraph/direct-download-links) to get the
latest Memgraph packages. 


```

File: pages/getting-started/install-memgraph/amazon-linux.mdx (hunk 4 above)
Comment (antejavor) [reply pages/getting-started/install-memgraph/amazon-linux.mdx:124]: Done.

File: pages/getting-started/install-memgraph/amazon-linux.mdx (hunk 3 above)
Comment (antejavor) [reply pages/getting-started/install-memgraph/amazon-linux.mdx]: This has been added to the latest commit.

**Comment by antejavor** [comment]
I have updated the PR.

**Review by kgolubic — APPROVED** [review]
Everything looks OK to me.

**Pull request merged**